client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,16 +44,20 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Support for task listeners in Camunda 8</h4>
-          Define user task listeners in your Camunda 8.7 diagrams, with helpful guidance provided every step of the way.
+          <h4>Element template runtime versions</h4>
+          Define the runtime version for each element template to ensure compatibility with the Camunda runtime.
         </li>
         <li>
-          <h4>Enhanced Variables panel</h4>
-          Quickly access the Variables panel from the application footer or toggle it using a keyboard shortcut.
+          <h4>Zeebe user task is now Camunda user task</h4>
+          Implement user tasks as Camunda user tasks to ensure full compatibility.
```

Comment:
Consider what we ship here:

* There is a clear path forward (Zeebe -> Camunda) user task first
* We now support you in the transition

I think this is not clearly reflected with the current phrase.

Reply:
Another try:

```
<h4>Migration to Camunda user tasks</h4>
Implement user tasks managed by Camunda with guidance on the way.
```